projects/core/src/occ/config-loader/occ-config-loader.module.ts

```diff
@@ -12,8 +13,15 @@ import { OccConfigLoaderService } from './occ-config-loader.service';
  */
 export function initConfig(
   configLoader: OccConfigLoaderService,
-  config: SiteContextConfig
-): ConfigInitializer {
+  config: SiteContextConfig,
+  /**
+   * TODO: remove it in 4.0
+   */
+  featureConfigService?: FeatureConfigService
+): ConfigInitializer | null {
+  if (featureConfigService && featureConfigService.isLevel('3.2')) {
```

Comment:
This legacy logic should be executed only for <3.2
```suggestion
  if (!featureConfigService.isLevel('3.2')) {
```